app/controllers/weeks_controller.rb

```diff
@@ -15,6 +15,27 @@ def show
         if (@rooms[0].state == "used" && @week.free_los(nil,current_user.id).count > 0)
           @rooms[0].update!(state: "do_not_use")
         end
+        # Ak sa nachadza viacero miestnosti na otvorenie necha len jednu alebo ziadnu ak uz nema nove learning objecty - boli odobrate
+        rooms_open = @rooms.select{|r| r.state == "do_not_use"}
+        if rooms_open.count > 1
+          j = rooms_open.count
+          if @week.free_los(nil,current_user.id).count == 0
+            j = j-1
+          else
+            j = j-2
+          end
+          for i in 0..j
```

Comment:
to bude asi trochu problem. Neviem, ci ti to Rails povoli. V tom pripade by som riesil takto:

```
if PODMIENKA_KED_SA_JEDNA_MIESTNOST_NEMENI
 rooms = rooms.where.not(id: rooms.first.id)
end
rooms.update_all(state: "used")
```